examples/basic/README.md

```diff
@@ -0,0 +1,201 @@
+<!-- Update the title -->
```

Comment:
An example should not contain all this info - just a very short summary as to what the examples demonstrates. For example, see module template https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/examples/basic/README.md